### PR TITLE
add: One click Fund issue 

### LINKED ISSUE #6149: One Click Fund Issue Link in Townsquare
### User Story

When I have a developer ready to work on an issue, I'd like a one click fund issue link as an option in the townsquare thread because this helps to streamline my workflow.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: No one click fund

### Current Behavior
No quick way to fund an issue 

### Expected Behavior
If thread contains github link, display ability to fund that issue via menu 
Open fund issue in new tab with url pre-populated in the form

### Definition of Done

- [ ] Anyone can one click fund an issue
- [ ] If thread contains github link, display ability to fund that issue via menu 
- [ ] Open fund issue in new tab with url pre-populated in the form
- [ ] Once the issue has been funded, show the comment in the original townsquare thread
- [ ] Post a video or screenshots for core team review

Screen Shot 2020-03-05 at 5 56 58 AM

### PR DESCRIPTION
**Pull request by developerfred**
##### Description

I added a button to create an issue if there is a problem and one to fund an issue that is on the box link.

##### Refers/Fixes
#6149

### CODE REVIEW

File: app/retail/templates/shared/activity.html [hunk 1]
```diff
@@ -245,6 +245,17 @@
             >
               <span class="action px-0 {% if row.flagged %}open{%endif%}"><i class="far fa-flag mr-2"></i></span> Report
             </a>
+
+            <a class="copy_activity mr-2 dropdown-item px-3 font-smaller-5" data-toggle="tooltip" title="create new issue about this townsquare message" href="https://github.com/gitcoinco/web/issues/new?title=TownSquare" data-url={{https://github.com/gitcoinco/web/issues/new?title=TownSquare?body=row.url}}>
+              <i class="fas fa-new mr-2"></i> Open new Issue
+            </a>
+
+            {% if row.metadata.resource.type == 'github' %}
+              <a class="copy_activity mr-2 dropdown-item px-3 font-smaller-5" data-toggle="tooltip" title="fund new issue about this" href="#" data-url={{'https://gitcoin.co/sync/get_issue_details?url=' + row.metadata.resource.type}}>
```
Comment (octavioamu) [inline app/retail/templates/shared/activity.html]: Maybe you want to do:
https://gitcoin.co/new?source=github issue url

**Comment by thelostone-mc** [comment]
@developerfred + @octavioamu Yo it seems like we've got a duplicate PR over at https://github.com/gitcoinco/web/pull/6150

I'm leaning towards that cause it also leaves a comment on the activity stating a bounty was funded which I felt was pretty darn neat

What do you guys think ?

**Comment by developerfred** [comment]
@thelostone-mc 
I suggest checking which is the best solution for the project. Regardless of the bounties or not.

**Comment by danlipert** [comment]
@thelostone-mc is this PR still valid? it seems to have a lot of unrelated changes

**Comment by thelostone-mc** [comment]
@danlipert + @developerfred so we'll end up going with https://github.com/gitcoinco/web/issues/6149

Thanks for the work @developerfred though ! 
Appreciate it <3

**Pull request closed without merging**